app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CommentsHelper
```

Comment:
What is the purpose of this empty module?

Reply:
This module also was created by default with the generate controller command:

rails generate controller CommandController Commands

Reply:
Same from here https://github.com/SanGeeky/rails-training/pull/1#discussion_r610291732.

Reply:
Answer: [#1 Comment about Helpers](https://github.com/SanGeeky/rails-training/pull/1#discussion_r610642205)